setup.py

```diff
@@ -90,6 +90,10 @@ def run(self) -> None:
                             from_location, os.stat(from_location).st_mode | stat.S_IEXEC
                         )
                     zip.write(from_location, to_location)
+            if platform == "mac":
+                shutil.copyfile(
+                    wheel_location, without_platform + "macosx_11_0_universal2.whl"
```

Comment:
My recollection is that Python won't accept identical wheel files for different platforms. Did you test this?